superchain/superchain.go

```diff
@@ -297,7 +297,6 @@ type AddressList struct {
 	OptimismPortalProxy               Address `json:"OptimismPortalProxy,omitempty" toml:"OptimismPortalProxy,omitempty"`
 	SystemConfigProxy                 Address `json:"SystemConfigProxy" toml:"SystemConfigProxy"`
 	ProxyAdmin                        Address `json:"ProxyAdmin" toml:"ProxyAdmin"`
-	SuperchainConfig                  Address `json:"SuperchainConfig,omitempty" toml:"SuperchainConfig,omitempty"`
```

Comment:
Did we check that there aren't any downstream dependencies that rely on this field? If there are, how should the access this field from Go?

Reply:
I can't find any references, I will double check by updating the imports in op-geth and the monorepo and seeing if we get any compiler errors.

Reply:
Found one reference, we would need to make this change in the monorepo https://github.com/ethereum-optimism/optimism/compare/gk/remove-superchain-config-from-scr?expand=1

Reply:
I did check `op-geth`, no changes required there. But see the other thread about whether we want to make this change overall.